src/libServer/IsolatedServer.cpp

```diff
@@ -264,8 +275,14 @@ Json::Value IsolatedServer::CreateTransaction(const Json::Value& _json) {
     if (!BlockStorage::GetBlockStorage().PutTxBody(tx.GetTranID(), twr_ser)) {
       LOG_GENERAL(WARNING, "Unable to put tx body");
     }
-
-    ret["TranID"] = tx.GetTranID().hex();
+    const auto& txHash = tx.GetTranID();
+    LookupServer::AddToRecentTransactions(txHash);
+    {
+      lock_guard<mutex> g(m_txnBlockNumMapMutex);
+      m_txnBlockNumMap[m_blocknum].emplace_back(txHash);
```

Comment:
Is this a temporary measure? `m_txnBlockNumMap` can potentially exhaust RAM.

Reply:
Yes, until we have a DB to store the txns, there is no other easier way